jbpm-bom/pom.xml

```diff
@@ -815,6 +815,12 @@
         <version>${version.org.kie}</version>
         <classifier>sources</classifier>
       </dependency>
+      <dependency>
+        <groupId>org.jbpm</groupId>
+        <artifactId>jbpm-wb-case-mgmt-showcase</artifactId>
+        <version>${version.org.kie}</version>
+        <type>war</type>
+      </dependency>
```

Comment:
@psiroky this showcase will be used for demonstrating the case management runtime capabilities. Although is not part of the workbench, it will be provisioned as a separated and optional war.